docs/ONCALL.adoc

```diff
@@ -0,0 +1,37 @@
+== Dates Oncall:
+
+=== Summary/Notes:
+
+* Use this section to put high level context that may be helpful to the next person on-call
+
+Example:
+
+* There was a deploy of a significant number of changes to `service` on `date`. This has triggered
+  a number of escalations which have exposed a lack of monitoring visibility into one of the systems
+  that `service` depends on. This ticket `T-1234` tracks investigating more metrics and this ticket
+  `T-4321` covers investigation into how the `service` could be made more robust in the future.
+
+=== Continuing Incidents
+
+* Put issues that weren't fully resolved here, include a link and a risk summary
+
+=== Incidents
+
+* Each incident/ticket that is worked on by the on-call should go in this section. This should include
+  a high level summary of the issue, what corrective action (if any) was taken, and a link for further details.
+** Summary - Action Taken - Ticket Link [Incident Link] - Quality of Life Impact - Exceptions
+** Options for each Section:
+*** Summary: one sentence description of the problem
+*** Action: Action Taken (Further Investigation Needed | Other, Ticket Link ) | No Action Taken (on-going maintenance ( inside | outside ) window | False Positive, Needs Threshold Adjustment)
```

Comment:
I still prefer my action categories. To me an action should be defined as something to resolve the _incident_ that the on-call responder was notified about. We are unable to differentiate between bad check scripts and bad/changing thresholds.